circle.yml

```diff
@@ -0,0 +1,11 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm install -g grunt-cli
+    - npm install grunt
+    - npm install grunt-contrib-qunit
+
+## Customize test commands
+test:
+  override:
+    - grunt test
```

Comment:
Does this generate any reports that we can put into CircleCI's [test results](https://circleci.com/docs/test-metadata/) or [artifacts](https://circleci.com/docs/build-artifacts/)?